lib/vsc/utils/generaloption.py

```diff
@@ -106,6 +106,22 @@ def check_str_list_tuple(option, opt, value):
         return klass(split)
 
 
+def get_empty_add_flex(allvalues):
+    """Return the empty element for add_flex action for allvalues"""
+    empty = None
+    emptyisNone = False # in case None is the empty value
```

Comment:
extensions of what?

don't include code you're not actually using, it looks silly